components/hardware/base/delfos/README.md

```diff
@@ -1,38 +1,60 @@
+# delfos
+
+`delfos` component is implemented as generic controller for omnidirectional robot (e.g omni wheels robot) in RoboComp left-handed coordinate. The follow list specifies set of implemented functions for controlling and robot state measuring as specified in interface `OmniRobot.idsl`:
```

Comment:
I think it's important to talk about the Left-handed coordinate system that Luis Vicente linked on his document:
https://www.evl.uic.edu/ralph/508S98/coordinates.htm
Not just mentionit  but explain in a paragraph how it works and the expected differences. It usually brings so much confusion when new people try to work with these components.